public/javascripts/gformFileUpload.js

```diff
@@ -27,22 +27,32 @@
     var submitButton = form.find(".govuk-button--secondary")
     var formComponentId = input.attr("id");
     var uploadedFiles = $("#" + formComponentId + "-files")
-    var frm = $("#gf-form");
     submitButton.css("display", "")
     submitButton.on("click", function(evt) {
       formGroup.hide()
       uploadedFiles.empty().append(startProgressBar());
       submitButton.css("display", "none")
       evt.preventDefault();
 
+      var gfForm = $("#gf-form");
       $.ajax({
-          type: frm.attr('method'),
-          url: frm.attr('action'),
-          data: frm.serialize()
-      }).then(function (){
-          frm.submit();
+          type: gfForm.attr("method"),
+          url: gfForm.attr("action"),
+          data: gfForm.serialize()
+      }).done(function (){
+          var upscanForm = $("#gf-upscan-"+formComponentId);
+          var data = new FormData(upscanForm[0]);
+
+          $.ajax({
```

Comment:
Hm, let's not do this second ajax call. This is getting complicated again. 

I think we should just click on `submitButton` programmatically (we need to change onClick callback prior to that obviously, as not to end up in a loop)